lib/iron_bank/local.rb

```diff
@@ -33,25 +33,33 @@ def reset_store
 
     private
 
+    # NOTE: Handle associations within the CSV export. E.g., when reading the
+    #       `ProductRatePlan.csv` file, we have `ProductRatePlan.Id` and
+    #       `Product.Id` fields. We want to end up with `id` and `product_id`
+    #       respectively.
+    def underscore_header
```

Comment:
Actually it cannot, because it's a closure. It needs to be executed _in the context of the class including the `Local` module_.

In the current version, `self.name # => IronBank::Resources::Product` (extending class).

I tried to make it a constant, and then `self.name # => IronBank::Local` which breaks the logic here.

Reply:
Ooooh. TIL 🍎